docs/sources/k6/next/shared/k6-operator/troubleshooting-how-to.md

```diff
@@ -38,21 +38,21 @@ In case of one `TestRun` Custom Resource (CR) creation with `parallelism: n`, th
    kubectl logs mytest-initializer-xxxxx
    ```
 
-If the Pods seem to be working but not producing an expected result and there's not enough information in the logs, you can use the k6 [verbose option](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/k6-options/#options) in the `TestRun` spec:
+#### `TestRun` with `cleanup` option
 
-```yaml
-apiVersion: k6.io/v1alpha1
-kind: TestRun
-metadata:
-  name: k6-sample
-spec:
-  parallelism: 2
-  script:
-    configMap:
-      name: 'test'
-      file: 'test.js'
-  arguments: --verbose
-```
+If a `TestRun` has [`spec.cleanup` option](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/executing-k6-scripts-with-testrun-crd/#clean-up-resources) set, as for example [`PrivateLoadZone`](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/private-load-zone/) tests always do, it may be harder to locate and analyze the Pod before it's deleted.
```

Comment:
```suggestion
If a `TestRun` has the [`spec.cleanup` option](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/executing-k6-scripts-with-testrun-crd/#clean-up-resources) set, as [`PrivateLoadZone`](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/private-load-zone/) tests always do, for example, it may be harder to locate and analyze the Pod before it's deleted.
```